lib/rdoc/generator/template/darkfish/table_of_contents.rhtml

```diff
@@ -1,4 +1,13 @@
 <body id="top" class="table-of-contents">
+<%= render '_sidebar_toggle.rhtml' %>
+
+<nav id="navigation" role="navigation">
+  <div id="project-navigation">
+    <%= render '_sidebar_navigation.rhtml' %>
+
+    <%= render '_sidebar_search.rhtml' %>
+  </div>
+</nav>
```

Comment:
I'm not sure this needs more content, but I needed a sidebar on this page, otherwise, it's harder to return to the Home page.

Reply:
I think it makes sense. And normally this page would have rich space on both sides, so adding it should be ok.